packages/core/admin/server/services/__tests__/auth.test.js

```diff
@@ -114,6 +114,17 @@ describe('Auth', () => {
     });
   });
 
+  describe('invalidatePassword', () => {
```

Comment:
From the way the other tests in that file were written, I'd suggest moving the case you added into the previous story (`validatePassword`), because the file creates the tests by the method. What do you think?